lib/reporter.js

```diff
@@ -108,6 +109,9 @@ class AllureReporter extends events.EventEmitter {
     }
 
     runnerCommand (command) {
+        if (!this.options.customStepsOnly) {
```

Comment:
It would be great if this option could get documented

Reply:
Added example to README